old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -128,38 +129,57 @@ The **geographic WGS84 lat/long** `CRS` has an origin - (0,0) -
 located at the intersection of the
 Equator (0° latitude) and Prime Meridian (0° longitude) on the globe.
 
-```{r, geographic-WGS84, echo=FALSE, message=FALSE, results='hide'}
+NOTE: The code for this map can be found in the .R document that is available for 
+download at the bottom of this page!
+
+```{r, geographic-WGS84, echo=FALSE, message=FALSE, results='hide', fig.cap="World map plotted on an X, Y grid with the axes labeled "longitude" and "latitude" and untis in degrees. The origin (0,0) is at the center of the plot."}
+
+#load libraries
+library(rgdal)
+library(ggplot2)
+
+# set working directory to ensure R can find the file we wish to import and where 
+# we want to save our files. Be sure to move the unzipped download into your working directory! 
+wd <- "~/Documents/data/" #This will depend on your local environment 
+setwd(wd)
+
+# read shapefile
+worldBound <- readOGR(dsn="ne_110m_land",
```

Comment:
@donal-at-NEON: I once again didn't add the `paste0` to these `readOGR`s (there are two more below). Did you try to run these without the `paste0` for the 04 tutorial? Let me know if that's the better way to do this and I'll add the `paste0` in.

Reply:
Yes, I did include the `paste0()` function in 04 (thanks for reminding me) e.g., `dsn=paste0(wd,"ne_110m_land/")` 
You can give that a try to see if it works for you - if not let me know and I can edit on my end

Reply:
Awesome! I'm curious as to why the trailing `/` worked for me and not for you, but I'm glad that we can be on the same page with `paste0()`
Thanks for making those updates